dianatanase/assignment3/Q1_tests.java

```diff
@@ -0,0 +1,92 @@
+import static org.junit.Assert.*;
+
+import java.util.Arrays;
+import java.util.HashSet;
+
+import org.junit.Test;
+
+
+public class Q1_tests {
+
+	// Set up the grid.
+	public char[][] gridSetUp(){
+		char[][] grid = new char[2][3];
```

Comment:
This should work as:

char[][] grid = new char[][] {
  {'a','a','r'},
  {'t','c','d'}
}